src/components/external-link/ExternalLink.tsx

```diff
@@ -1,15 +1,22 @@
 import React from "react";
 import { ExternalLinkAltIcon } from "@patternfly/react-icons";
+import { Button } from "@patternfly/react-core";
 
 export const ExternalLink = ({
   title,
   href,
   ...rest
 }: React.HTMLProps<HTMLAnchorElement>) => {
```

Comment:
You'll have to change this to:
```suggestion
}: ButtonProps) => {
```